Wireframe/style.css

```diff
@@ -45,14 +45,19 @@ svg {
 Setting the overall rules for page regions
 https://www.w3.org/WAI/tutorials/page-structure/regions/
 */
-main {
+*main {
   max-width: var(--container);
   margin: 0 auto calc(var(--space) * 4) auto;
 }
 footer {
   position: fixed;
   bottom: 0;
   text-align: center;
+  color: white;
```

Comment:
Your footer is invisible when I scroll down and it sits in front of the white background. Can you you adjust the color of the font (or the background color) of the footer to make your footer visible no matter where I scroll?